girder/test/test_cache.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2014 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import pytest
+
+from girder import _setupCache
+from girder.constants import SettingKey
+from girder.models.setting import Setting
+from girder.utility import config
+
+import mock
+
+
+@pytest.fixture
+def server(server):
+    # Extend server fixture to enable the cache
+    cfg = config.getConfig()
+    cfg['cache']['enabled'] = True
+    _setupCache()
+
+    yield server
+
+
+def testCachesAreAlwaysConfigured():
+    from girder.utility.cache import cache, requestCache
+
+    assert cache.is_configured
+    assert requestCache.is_configured
```

Comment:
I'd feel more comfortable if this was `assert cache.is_configured is True`; just using the truthiness value could allow for errors, for instance if `is_configured` were actually a function that is supposed to be called. (Even if it's not a function, it's best to maintain this precedent of explicitness.)

Reply:
Done.